client/src/components/LoggedIn.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
Just a nit and nothing that needs to be fixed now, but a more helpful name for the component would be useful. Don't really know what `LoggedIn` would do, just looking at the code base. Maybe something like `getZipCode` or w/e

Reply:
LoggedIn is just a placeholder for now.  I'd like to leave it as is for now with the proviso that anyone can change it at will.
I spoke with Chris and Michael for ex who might want to use it for their join session, create session page.